test/e2e/utils.go

```diff
@@ -580,3 +583,16 @@ func getOtelCollectorOptions() map[string]interface{} {
 
 	return otelOptions
 }
+
+func logWarningEvents() {
+	eventList, err := fw.KubeClient.CoreV1().Events(namespace).List(metav1.ListOptions{})
+	require.NoError(t, err)
+	if len(eventList.Items) > 0 {
+		logrus.Infof("Events for test %s", t.Name())
+	}
+	for _, event := range eventList.Items {
+		if event.Type != "Normal" {
+			logrus.Warnf("Type: %s Reason: %s Message: %s", event.Type, event.Reason, event.Message)
```

Comment:
That sounds really cool! How noisy is it? Can you paste an example?

Reply:
If we stick to warning events it shouldn't be too noisy, unless something is really broken.  There can be many Normal events, which is why I have excluded those.